keadm/cmd/keadm/app/cmd/reset.go

```diff
@@ -67,15 +67,16 @@ func NewKubeEdgeReset(out io.Writer, reset *common.ResetOptions) *cobra.Command
 		Long:    resetLongDescription,
 		Example: resetExample,
 		PreRunE: func(cmd *cobra.Command, args []string) error {
-			whoRunning, err := util.IsCloudCore()
+			whoRunning, err := util.RunningModule()
 			if err != nil {
 				return err
 			}
 			switch whoRunning {
 			case common.KubeEdgeEdgeRunning:
 				IsEdgeNode = true
 			case common.NoneRunning:
-				return fmt.Errorf("None of KubeEdge components are running in this host")
+				fmt.Println("None of KubeEdge components are running in this host, exit")
+				os.Exit(0)
```

Comment:
Has any difference with `return fmt.Errorf`?

Reply:
If return err, `keadm` would print debug stack and help info which becomes confused for users.

Old logic would detect service exists or not, in fact we just consider binary running or not here, so simplify logic would fix the issue.